python/ray/autoscaler/_private/util.py

```diff
@@ -107,8 +107,8 @@ def fillout_defaults(config: Dict[str, Any]) -> Dict[str, Any]:
     except Exception:
         # We don't want to introduce new errors with filling available node
         # types resources feature.
-        logger.exception("Failed to autodetect node resources")
-
+        logger.exception("Failed to autodetect node resources.")
+        raise
```

Comment:
Actually, didn't we explicitly not re-raise this exception before? Why re-raise it now?

Reply:
Didn't we agree that the instance should be available in the instance types? And if a value error was raised the used should fix the node type.

Reply:
> lgtm, out of curiosity, would it be possible/easy to test this?

BTW, I realized this already tests it:
`python/ray/tests/test_autoscaler.py::AutoscalingTest::testValidateDefaultConfig`